### DISCUSSION TITLE
SSB audio macros - add delay after playback ends

### DISCUSSION THREAD

**Discussion by trapeznikovk** [mbridak/not1mm#27] [Ideas]
https://github.com/mbridak/not1mm/assets/10850583/81d7f94e-e461-4957-a0b3-76af9419348d

Hello,
I tried to play SSB audio macros on my Ermak rig under Linux. There is a dedicated audio loopback device and an EAX utility to send audio data to TRX over a UDP channel. It works well (see attachment). But it looks like we need to add a customized delay (up to ~300ms) after the playback ends and switches to RX 
73,
UT2UK
Kons

**Comment by mbridak** [comment]
That's a very handsome rig.
I'll see what I can do with the delay.

**Comment by mbridak** [comment]
I believe this may be resolved with code pushed today.
